src/ResidentDirectory.js

```diff
@@ -5,21 +5,20 @@ import AddResident from './AddResident.js'
 import Card from 'react-bootstrap/Card';
 import Button from 'react-bootstrap/Button';
```

Comment:
@aashnasheth 
See this for reference

I do believe that you can import multiple together as import { Card, Button } from 'react-bootstrap'; as well